stream-chat-android-client/src/androidTest/java/io/getstream/chat/android/client/poc/ClientInstrumentationTests.kt

```diff
@@ -101,8 +100,8 @@ internal class ClientInstrumentationTests {
             }
             client.subscribe(connectedEventConsumer::onEvent)
         }.andThen {
-            await().atMost(5, SECONDS).until(setUserListener::onSuccessIsCalled)
-            await().atMost(5, SECONDS).until(connectedEventConsumer::isReceived)
+            await().atMost(50, SECONDS).until(setUserListener::onSuccessIsCalled)
+            await().atMost(50, SECONDS).until(connectedEventConsumer::isReceived)
```

Comment:
This seems like a super long time to wait in this test, does it really take this long? We'd be better off leaving this test ignored if it does.

Reply:
For now we are not running those tests in CI, they are instrumentation tests that aren't configured into our CI